### ISSUE TITLE
Include the help files (markdown) with the installer.

### ISSUE THREAD

**Issue by LordDurus** [QB64Official/vscode#108]
Include the help files (markdown) with the installer.

This will make setup for new users easier.
The existing setting will be used, this way if someone wants to use a custom path for the help they can.

**Comment by LordDurus** [comment]
Need a good way of updating the help files.  Currently I'll have to remember to do it.

**Comment by LordDurus** [comment]
These are the qb64 help files.  Used for the F1 help and hovertext.  Including them with the extension gives new users one less thing to setup.  

The extension still uses the helpPath setting.  It will get defaulted to the markdown files included with the extension.  But it can be changed to the QB64 folder.   If the helpPath is already set it won't get changed.   The setting will be already set for anyone who already has working hovertext/f1.  

For now I just copied the files in since they don't change very much.

Having a process that pulls them down at build would be better.

I did find an issue while I was in there.  If the case of the keywork in the code didn't match the case of the filename.  Then the hovertext would show on Linux.  I fixed that.

I'm going to create a different issue for this, what files are needed to QB64 IDE running?  I'd like to include that as well.  Then someone would only need to install the extension to start coding.
